src/masonite/validation/Validator.py

```diff
@@ -385,7 +385,8 @@ def negated_message(self, attribute):
 
 
 class length(BaseValidation):
-    def __init__(self, validations, min=1, max=999999, messages={}, raises={}):
+
+    def __init__(self, validations, min=1, max=False, messages={}, raises={}):
```

Comment:
@girardinsamuel I modified the script here a bit. Instead of making max a number we can just set it to False and then checked the logical value in the message methods below